tests/admin_changelist/tests.py

```diff
@@ -1636,8 +1635,7 @@ def test_no_user(self):
         """{% get_admin_log %} works without specifying a user."""
         user = User(username="jondoe", password="secret", email="super@example.com")
         user.save()
-        ct = ContentType.objects.get_for_model(User)
-        LogEntry.objects.log_action(user.pk, ct.pk, user.pk, repr(user), 1)
+        LogEntry.objects.log_actions(user.pk, [user], 1, "", True)
```

Comment:
```suggestion
        LogEntry.objects.log_actions(user.pk, [user], 1, single_object=True)
```

Reply:
I think that the new `log_actions()` method should have `single_object` as a keyword-only argument. That'll avoid things like this.